src/main/java/org/sagebionetworks/bridge/exporter/util/BridgeExporterUtil.java

```diff
@@ -82,6 +62,11 @@
      * @return the schema key associated with that record
      */
     public static UploadSchemaKey getSchemaKeyForRecord(Item record) {
+        if (!record.hasAttribute("schemaId") || !record.hasAttribute("schemaRevision")) {
+            // This is a schemaless record.
+            return null;
+        }
```

Comment:
It would be nice during validation if we find schemaId without schemaRevision or surveyGuid without surveyCreatedOn, or vice versa, we logged a warning or something. It's good to persist the upload but having partial information seems to suggest that they are trying to use a schema but don't fully understand our system.

Reply:
Going forward, schemaless will be the norm, not the exception. It wouldn't be "partial information", nor would it indicate that researchers "don't fully understand our system". It will be a fully supported first-class citizen and should be the default way our system is used.

With that in mind, I don't believe it's worth logging a warning over.

Reply:
Not if both are missing (this is normal, yes), but if one of the two values but not both are present.